spec/spec_helper.rb

```diff
@@ -1,6 +1,21 @@
 require 'chefspec'
 require 'chefspec/berkshelf'
 
+CENTOS_7 = {
+  platform: 'centos',
+  version: '7',
+}.freeze
+
```

Comment:
Please remove CentOS 7 as we're not going to worry about that. We should remove that from our generator template.